src/backend/columnar/sql/downgrades/columnar--10.0-1--9.5-1.sql

```diff
@@ -0,0 +1,45 @@
+/* columnar--10.0-1--9.5-1.sql */
+
+SET search_path TO cstore;
+
+DO $proc$
+BEGIN
+
+IF version() ~ '12' or version() ~ '13' THEN
```

Comment:
it seems quite likely that one of these strings may at some point appear in version() even on PostgreSQL 11